soccer/NewRefereeModule.cpp

```diff
@@ -179,12 +186,12 @@ void NewRefereeModule::run() {
         ballPlacementx = packet->wrapper.designated_position().x();
         ballPlacementy = packet->wrapper.designated_position().y();
 
-        std::string blue_name = blue_info.name;
-        for (char& letter : blue_name) {
+        std::string yellow_name = yellow_info.name;
+        for (char& letter : yellow_name) {
             letter = tolower(letter);
         }
 
-        blueTeam(blue_name == "robojackets");
+        blueTeam(yellow_name != Team_Name_Lower);
```

Comment:
yellow_team is set by the referee, and that's how we switch teams in comp mode. If the referee fails to set us, we'll get the wrong team though.